lms/djangoapps/courseware/access.py

```diff
@@ -100,6 +104,8 @@ def has_access(user, action, obj, course_key=None):
         return _has_access_descriptor(user, action, obj, course_key)
 
     if isinstance(obj, CourseKey):
+        if isinstance(obj, CCXLocator):
+            obj = obj.to_course_locator()
```

Comment:
Should there instead be a  `_has_access_ccx_key`, which for the moment simply strips off the CCX part of the locator and passes down to `_has_access_course_key`?